mobile/library/common/system/default_system_helper_apple.cc

```diff
@@ -19,4 +21,8 @@ std::vector<std::pair<int64_t, ConnectionType>> DefaultSystemHelper::getAllConne
   return {};
 }
 
+void DefaultSystemHelper::bindSocketToNetwork(Network::ConnectionSocket&, int64_t) {
+  PANIC("unreachable");
```

Comment:
why panicking here for Apple? wouldn't this get called by platform_packet_writer_factory.cc on any platform?

Reply:
2 choice here: 1) Apple will continue using the default packet writer. It doesn't propagate the network handle to QUICHE anyway. 2) using the extension packet writer as well, but because of lack of network handle, this function won't be called.